### PR TITLE
Pa02

### PR DESCRIPTION
**Pull request by petterho**
Our solution to Pa02

### CODE REVIEW

**Review by jkorsvik — APPROVED** [review]
Looks good Peter! 💃

File: src/pa02/snakes_simulations.py [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+        """
+        num_of_turns = 0
+        if self.randomize:
+            shuffle(self.player_field)
+
+        players = []
+        for player in self.player_field:
+            players.append(player(self.board))
+
+        while True:
```
Comment (danielambrosius) [inline src/pa02/snakes_simulations.py]: maybe `while not self.goal_reached()` might be better?

File: src/pa02/snakes_simulations.py [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+        if ladders is not None:
+            self.ladders = ladders
+        else:
+            self.ladders = [(1, 40), (8, 10), (36, 52), (43, 62),
+                            (49, 79), (65, 82), (68, 85)]
+        if chutes is not None:
+            self.chutes = chutes
+        else:
+            self.chutes = [(24, 5), (33, 3), (42, 30), (56, 37),
+                           (64, 27), (74, 12), (87, 70)]
```
Comment (danielambrosius) [inline src/pa02/snakes_simulations.py]: Would reccommend tranforming these to dictionaries. The input argument type does not dictate the datatype used internally.

**Review by danielambrosius — APPROVED** [review]
Looks alright. approved

**Pull request merged**